src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir-Book, Avenir-Heavy, sans-serif, serif;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.big {
+  padding-bottom: 20px;
+  position: relative;
+}
+
+.small {
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  padding-top: 0;
+  outline: none;
+  font-family: inherit;
+}
+
+.big_search {
+  height: 70px;
+  padding-left: 62px;
+}
+
+.img {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.small_search {
+  height: 42px;
+  position: relative;
+  padding-left: 33px;
+}
+
+.img_small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+  position: absolute;
+}
+
+.big_search::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.small_search::placeholder {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.big_search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: Avenir-Book, Avenir-Heavy, sans-serif, serif;
+}
+
+.big_search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: Avenir-Book, Avenir-Heavy, sans-serif, serif;
+}
+
+.small_search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  font-family: Avenir-Book, Avenir-Heavy, sans-serif, serif;
+}
+
+.small_search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-size: 14px;
+  line-height: 19px;
+  font-family: Avenir-Book, sans-serif, serif;
```

Comment:
```suggestion
}

.small_search:focus {
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  border: 1px solid #e1e7ed;
  border-radius: 3px;
  font-size: 14px;
  line-height: 19px;
```

Reply:
In the template on Figma, the font is specified
font-family: 'Avenir';
font-size: 16px;
line-height: 22px;